pquery-run.sh

```diff
@@ -1388,24 +1399,37 @@ pquery_test(){
           else
             echoit "Loading metadata from ${WORKDIR}/step_$((${TRIAL}-1)).dll ..."
           fi
+          if [[ ${PXC} -eq 0 && ${GRP_RPL} -eq 0 ]]; then
+            CMD="${PQUERY_BIN} --database=test --threads=${THREADS} --queries-per-thread=${QUERIES_PER_THREAD} --logdir=${RUNDIR}/${TRIAL} --user=root --socket=${SOCKET} --seed ${SEED} --step ${TRIAL} --metadata-path ${WORKDIR}/ --seconds ${PQUERY_RUN_TIMEOUT} ${DYNAMIC_QUERY_PARAMETER}"
+            echoit "$CMD"
+            $CMD >${RUNDIR}/${TRIAL}/pquery.log 2>&1 &
+            PQPID="$!"
```

Comment:
echoit "$CMD"
             $CMD >${RUNDIR}/${TRIAL}/pquery.log 2>&1 &
             PQPID="$!"

are common and can be moved after else

Reply:
Done